build.sbt

```
@@ -41,7 +41,7 @@ lazy val root = Project("sbt-scoverage", file("."))
         case "2.12" =>
           (pluginCrossBuild / sbtVersion).value
         case _ =>
-          "2.0.0-M3"
+          "2.0.0-M4"
```

shall we update to 2.0.0-RC3 already, instead of merging this PR and then doing another PR for 2.0.0-RC3?
```suggestion
          "2.0.0-RC3"
```